src/main/java/com/simpligility/maven/plugins/android/phase09package/ApkMojo.java

```diff
@@ -825,7 +825,7 @@ private File removeDuplicatesFromJar( File in, List<String> duplicates,
                     //if not handled by transformer, add (once) to duplicates jar
                     if ( !resourceTransformed )
                     {
-                        if ( !duplicatesAdded.add( entry.getName() ) )
+                        if ( duplicatesAdded.add( entry.getName() ) )
```

Comment:
Of course.
Can I suggest you change it to make the meaning clearer.
The efficiency gain it gives is of low value here.